### PR TITLE
Update to support multiple context types.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Review by JSAssassin — APPROVED** [review]

File: lib/schemas/operationContext.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
- * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2019-2021 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
 module.exports = {
   type: 'string',
-  enum: ['https://w3id.org/security/v2']
+  enum: [
+    'https://w3id.org/security/v2'
```
Comment (mattcollier) [inline lib/schemas/operationContext.js:9]: Can this be pulled in from `security-context` as well?

File: package.json [hunk 2]
```diff
@@ -28,15 +28,14 @@
...
     "bs58": "^4.0.1",
-    "http-signature-zcap-verify": "^4.0.0",
-    "jsonld-signatures": "^7.0.0",
-    "ocapld": "^2.0.0"
+    "http-signature-zcap-verify": "^6.0.0",
+    "security-context": "^4.0.0"
+  },
+  "devDependencies": {
+    "eslint": "^7.19.0",
+    "eslint-config-digitalbazaar": "^2.6.1"
```
Comment (mattcollier) [inline package.json:39]: `http-signature-zcap-verify` has a node>=12 requirement, need an engine specified here as well.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # webkms-switch ChangeLog
 
+## 4.0.0 - 
+
+### Changed
+- **BREAKING**: Use latest version of `http-signature-zcap-verify`.
```
Comment (mattcollier) [inline CHANGELOG.md:6]: Maybe subitems:
- Only support `Ed25519Signature2020` proofs.
- Node.js >= 12 is required
- https://github.com/digitalbazaar/http-signature-zcap-verify/blob/main/CHANGELOG.md

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: lib/schemas/operationContext.js (hunk 1 above)
Comment (JSAssassin) [reply lib/schemas/operationContext.js]: addressed here c3e1c23

File: package.json (hunk 2 above)
Comment (JSAssassin) [reply package.json:39]: addressed c3e1c23

File: CHANGELOG.md (hunk 3 above)
Comment (JSAssassin) [reply CHANGELOG.md:6]: Updated c3e1c23

**Review by mattcollier — APPROVED** [review]

**Pull request merged**